exercises/exercise3.js

```diff
@@ -1,12 +1,444 @@
+      const state = params.state;
+      if (
+        state.toLowerCase() === "georgia" ||
+        state.toLowerCase() === "california" ||
+        state.toLowerCase() === "washington"
+      ) {
+        return (
+          "OK Nice, I love " +
+          state +
+          "! " +
+          state +
+          " people are my people. Are you ready to start playing?"
+        );
+      } else if (state.toLowerCase() === "florida") {
+        return "What is up with you people? I have so many questions. I just don't think we can be friends, goodbye.";
+      } else if (
+        state.toLowerCase() === "alabama" ||
+        state.toLowerCase() === "alaska" ||
+        state.toLowerCase() === "arizona" ||
+        state.toLowerCase() === "arkansas" ||
+        state.toLowerCase() === "colorado" ||
+        state.toLowerCase() === "connecticut" ||
+        state.toLowerCase() === "delaware" ||
+        state.toLowerCase() === "hawaii" ||
+        state.toLowerCase() === "idaho" ||
+        state.toLowerCase() === "illinois" ||
+        state.toLowerCase() === "indiana" ||
+        state.toLowerCase() === "iowa" ||
+        state.toLowerCase() === "kansas" ||
+        state.toLowerCase() === "kentucky" ||
+        state.toLowerCase() === "louisiana" ||
+        state.toLowerCase() === "maine" ||
+        state.toLowerCase() === "maryland" ||
+        state.toLowerCase() === "massachusetts" ||
+        state.toLowerCase() === "michigan" ||
+        state.toLowerCase() === "minnesota" ||
+        state.toLowerCase() === "mississippi" ||
+        state.toLowerCase() === "missouri" ||
+        state.toLowerCase() === "montana" ||
+        state.toLowerCase() === "nebraska" ||
+        state.toLowerCase() === "nevada" ||
+        state.toLowerCase() === "new hampshire" ||
+        state.toLowerCase() === "new jersey" ||
+        state.toLowerCase() === "new mexico" ||
+        state.toLowerCase() === "new york" ||
+        state.toLowerCase() === "north carolina" ||
+        state.toLowerCase() === "north dakota" ||
+        state.toLowerCase() === "ohio" ||
+        state.toLowerCase() === "oklahoma" ||
+        state.toLowerCase() === "oregon" ||
+        state.toLowerCase() === "pennsylvania" ||
+        state.toLowerCase() === "rhode island" ||
+        state.toLowerCase() === "south carolina" ||
+        state.toLowerCase() === "south dakota" ||
+        state.toLowerCase() === "tennessee" ||
+        state.toLowerCase() === "texas" ||
+        state.toLowerCase() === "utah" ||
+        state.toLowerCase() === "vermont" ||
+        state.toLowerCase() === "virginia" ||
+        state.toLowerCase() === "west virginia" ||
+        state.toLowerCase() === "wisconsin" ||
+        state.toLowerCase() === "wyoming"
+      ) {
+        return "I think we'll be OK. Are you ready to start playing?";
+      } else {
+        return "You don't spell so good do you? Please try again.";
+      }
+    },
+    isListening: true,
+  },
```

Comment:
Haha, this is delightful. Well done on your syntax here - despite being a lot of code it is clean and very clear what is going on here.